### ISSUE TITLE
need to remove spaces from IBAN before searching

### ISSUE THREAD

**Issue by jeffallen** [akretion/odoo-usability#32]
In this file:
https://github.com/akretion/odoo-usability/blob/8.0/account_bank_statement_import_usability/account_bank_statement_import.py

account_number needs to be canonicalized the same way before you substitute it into the SQL query. Otherwise a search for an IBAN with spaces in it will not match, even if the IBAN in the database had spaces, because you've removed the spaces from the IBAN in the database.

I had to fix this for a client and I thought you'd like to know about it here as well.

**Comment by alexis-via** [comment]
Yes, you are right. I haven't seen a bank statement file with spaces in the account number thought, that's why we haven't been bothered by that yet.
